dev-client/android/app/build.gradle

```diff
@@ -170,3 +170,13 @@ dependencies {
 }
 
 apply from: file("../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesAppBuildGradle(project)
+
+// NOTE: This is to add vector icons to our app
+// We can add more here if we ever needed to, but as we are using the Material UI, probably won't need to
+// See https://github.com/oblador/react-native-vector-icons#android
+// REVIEW: Will need to do the same process with iOS
```

Comment:
I want to make an issue for dealing with these kinds of chores. I think we should have a process for how to deal with this stuff.

Reply:
in https://github.com/techmatters/terraso-web-client/pull/934 i am keeping a list of issues to file in the PR itself, and i will file them as chore issues just before merging, does that seem reasonable?

Reply:
Yes, good idea Garo.